modules/administration_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -528,8 +528,7 @@ PHP module missing. | There will be a white page/exception in web UI. It
 will be a full page issue, and the application will not be usable
 
 | All enabled, but the Redis server is not running | The application will
-be usable. But any file operation will return a *``500 Redis went
-away''* exception
+be usable. But any file operation will return a "500 Redis went away" exception
```

Comment:
why do we have standard double quotes here where on all the other examples we use single backticks?

Reply:
Because we don't have a written style guide.